### ISSUE TITLE
Import-Module Microsoft.Graph.DirectoryObjects

### ISSUE THREAD

**Issue by William-Francillette** [microsoft/Microsoft365DSC#2713]
#### Details of the scenario you tried and the problem that is occurring
I am developing a resource and query Get-MgDirectoryObject from Microsoft.Graph.DirectoryObjects but receive an error:

```
The 'Get-MgDirectoryObject' command was found in the module 'Microsoft.Graph.DirectoryObjects', but the module could not be loaded. For more information, run 'Import-Module Microsoft.Graph.DirectoryObjects'.
    + CategoryInfo          : ObjectNotFound: (Get-MgDirectoryObject:) [], CimException
    + FullyQualifiedErrorId : CouldNotAutoloadMatchingModule
    + PSComputerName        : localhost
```

Do I need to import the module or should that be loaded automatically with the M365DSC module?

If I need to import the module manually, what will be the best place to import it?
I am using the cmdlet in Set-TargetResource

I'm using PS 5.1 and have already installed the module 

Thanks
Will
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# insert configuration here
```

**Comment by andikrueger** [comment]
At the moment, this module is not part of the various dependencies defined in this file:

https://github.com/microsoft/Microsoft365DSC/blob/Dev/Modules/Microsoft365DSC/Dependencies/Manifest.psd1

m365dsc loss all modules that are defined in there. If this a requirement for a resource: it should be added to the dependencies. 

Could you still run import-module to get a proper error message, why this module cannot be loaded automatically?

**Comment by William-Francillette** [comment]
Yep I am using the cmdlet for a resource

My issue is resolved - I had version 1.19.0 of Microsoft.Graph.DirectoryObjects  installed but all the other modules were on 1.17.0 (specifically authentication)

After installing 1.17.0 and restarting my ISE it was working

**Comment by andikrueger** [comment]
Good find! 1.19 is the current version with the latest m365dsc release. 

Please make sure to update to the current graph modules ;)
